### PR TITLE
Issue #1764: Switch to P2 modules

### PR DESCRIPTION
**Pull request by cableman**
se http://platform.dandigbib.org/issues/1764

### CODE REVIEW

File: ding2.install [hunk 1]
```diff
@@ -369,3 +369,17 @@ function ding2_update_7019() {
...
+ */
+function ding2_update_7021() {
+  module_disable(array('ding_bookmark'));
+}
+
+/**
+ * Enable metatag and metatag_panels modules.
+ */
+function ding2_update_7022() {
+  module_enable(array('ding_p2_installer'));
```
Comment (kasperg) [inline ding2.install]: The doc block does not seem to match the content?

**Comment by kasperg** [comment]
@cableman: Review complete. I have a single comment.

File: ding2.install (hunk 1 above)
Comment (cableman) [inline ding2.install]: Your are right... it was some fast coding :-) Sorry have made a new commit.

**Comment by kasperg** [comment]
Reviewed, approved and merged. Thanks!

**Pull request merged**